node/src/main/scala/coop/rchain/node/web/Transaction.scala

```
@@ -149,7 +149,7 @@ final case class CacheTransactionAPI[F[_]: Concurrent](
   private val blockDeferMap: TrieMap[String, Deferred[F, TransactionResponse]] = TrieMap.empty
 
   def getTransaction(blockHash: String): F[TransactionResponse] =
-    store.get1(blockHash) >>= { transactionOpt =>
+    (if (blockHash.isEmpty) None.pure else store.get1(blockHash)) >>= { transactionOpt =>
```

1. What will be executed when block hash is empty? Do we want to execute anything?
1. Is it the same to not have transaction for some input or exit early because of invalid input (which cannot have transactions at all)?

NOTE: The purpose of this PR is not to hide the error.